draft-ietf-privacypass-architecture.md

```diff
@@ -210,23 +210,28 @@ contexts:
 
 Redemption context:
 : The interactions and set of information shared
-between the Client and Origin. This context includes all information
-associated with redemption, such as the timestamp of the event, Client
-visible information (including the IP address), and the Origin name.
+between the Client and Origin, i.e., the information that is provided or
+otherwise available to the Origin during redemption that might be used
+to identify a Client. This context includes all information associated
+with redemption, such as the timestamp of the event, Client visible
+information (including the IP address), and the Origin name.
 
 Issuance context:
-: The interactions and set of information shared
-between the Client, Attester, and Issuer. This context includes all
-information associated with issuance, such as the timestamp of the event,
-any Client visible information (including the IP address), and the
-Origin name (if revealed during issuance).
+: The interactions and set of information shared between the Client, Attester,
+and Issuer, i.e., the information that is provided or otherwise available
+to Attester and Issuer during issuance that might be used to identify a Client.
+This context includes all information associated with issuance, such as the
+timestamp of the event, any Client visible information (including the IP
+address), and the Origin name (if revealed during issuance).
```

Comment:
This does not include the challenge, right?  It includes configuration information from the challenge (the type of issuance protocol), but not the challenge content itself, which needs to remain secret.